rotkehlchen/chain/substrate/manager.py

```diff
@@ -132,108 +119,111 @@ def __init__(
         https://guide.kusama.network/docs/en/kusama-index
         https://wiki.polkadot.network/en/
 
+        External Address Format (SS58) documentation:
+        https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)
+
         Polkascan py-scale-codec:
         https://github.com/polkascan/py-scale-codec/tree/master
 
         Polkascan py-substrate-interface:
         https://github.com/polkascan/py-substrate-interface
         https://polkascan.github.io/py-substrate-interface/base.html
 
-        Postman Substrate Interface API:
-        https://documenter.getpostman.com/view/9969999/SWT5iLXH?version=latest
-
         Subscan API documentation:
         https://docs.api.subscan.io
         """
-        if chain not in Chain:
+        if chain not in SubstrateChain:
             raise AttributeError(f'Unexpected SubstrateManager chain: {chain}')
 
-        log.debug(f'Initializing substrate {chain} manager')
+        log.debug(f'Initializing {chain} manager')
         self.chain = chain
         self.greenlet_manager = greenlet_manager
         self.msg_aggregator = msg_aggregator
         self.available_node_attributes_map: DictNodeNameNodeAttributes = {}
-        self.chain_properties: ChainProperties
-        self._connect()
+        self.chain_properties: SubstrateChainProperties
+        if connect_at_start:
```

Comment:
A bit nitpicky, but I really don't like checking length of list as Truthy value in python. Please check explicitly as len(connect_at_start) == 0.

The reason is readability. When I first read the code I thought it was a boolean. Only by looking around do I understand that it's a truthy check on a list.